apps/accounts/validators.py

```diff
@@ -96,14 +96,11 @@ def __init__(self,
                "but having password_expire({}) >= password_reuse_interval({})"
 
         check_opt_err = []
-        if password_min_age > 0 and password_reuse_interval > 0 \
-                and password_min_age > password_reuse_interval:
+        if password_min_age > 0 and 0 < password_reuse_interval < password_min_age:
```

Comment:
I think this can be further simplified to
```suggestion
        if 0 < password_reuse_interval < password_min_age:
```